### PR TITLE
Participant selection

### PR DESCRIPTION
**Pull request by dhuppenkothen**
Hi all, here's a first draft of the participant selection section. 

Let me know what you think! It's still ab it rough around the edges (feel free to edit! I'm totally not used to writing a paper like this one!).

I'm thinking it's probably too long/detailed. If it can be shortened, let me know, and I'll stick whatever gets cut out into the entrofy paper. :)

Also part of this PR: I've merged the "audience" and "participant selection" sections into one, because I found no good way to break them up, and I reorganized a few parts (it seemed to make more sense to talk about what a hackathon is before talking about why we need one).

I don't think there's a huge amount of duplication in the previous text, actually, and what duplication there is seems to be just mentioned first, and then expanded upon later, so I think it might actually be all right like this for now.

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -33,48 +33,62 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
...
+A disadvantage of the hackathon model is the common perception (whether accurate or not) that the week is designed for experts in technical tools, which may discourage others from attending.
+In 2014, we started Astrohack week to try to fill the gaps between these models. The hack week model combines pedagogy (often focused on statistical and computational techniques) with room for collaborative hacks or creative projects, with the goal of encouraging collaboration and learning among people at various stages of their career.
 
 ![alt text](fig/HackSpectrum.png "Comparison of Extracurricular Workshop Models")
 
-As of the publication of this paper, we have run five such HackWeek events: three focused on Astronomy, one focused on Neuroscience, and one focused on Geophysics.
-Below we will share some of the philosophy behind the HackWeek model, practical lessons we have learned in organizing these events, and recommendations for future HackWeeks in other disciplines.
+As of the publication of this paper, we have run five such hack week events: three focused on Astronomy, one focused on Neuroscience, and one focused on Geophysics.
```
Comment (arokem) [inline paper.md:43]: We'll need to increment these numbers, if this gets published past September 😄

File: paper.md [hunk 2]
```diff
@@ -33,48 +33,62 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
...
-A disadvantage of the hackathon model is the common perception (whether accu rate or not) that the week is designed for experts in technical tools, which may discourage others from attending.
-In 2014, we started AstroHackWeek to try to fill the gaps between these models. The HackWeek model combines pedagogy (often focused on statistical and computational techniques) with room for collaborative hacks or creative projects, with the goal of encouraging collaboration and learning among people at various stages of their career.
+A disadvantage of the hackathon model is the common perception (whether accurate or not) that the week is designed for experts in technical tools, which may discourage others from attending.
+In 2014, we started Astrohack week to try to fill the gaps between these models. The hack week model combines pedagogy (often focused on statistical and computational techniques) with room for collaborative hacks or creative projects, with the goal of encouraging collaboration and learning among people at various stages of their career.
```
Comment (arokem) [inline paper.md]: Really? Two words? OK...

**Comment by karthik** [comment]
Thanks @dhuppenkothen!
Will review this PR by Tuesday next week and update this thread.

**Comment by dhuppenkothen** [comment]
So this is no longer just participant selection, since I kept writing and didn't pay attention what branch I was on. But I think it's nicer to keep the draft together anyway.

It's still pretty rough, since it's not a type of scientific writing I'm used to doing, but comments are always welcome. :)

File: paper.md [hunk 3]
```diff
@@ -33,120 +33,183 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
...
+The need to carefully select participants dovetails with another requirement: transparency. Up to date, all hack weeks we have organized have been oversubscribed by at least a factor of two (IS THIS TRUE?), and thus necessarily require turning applicants away. The selection procedure should be as transparent to the applicants as possible, and the organizing committee should be held accountable for their performance. Transparency is necessary for applicants to understand acceptance/rejection decisions, and accountability is of crucial importance for the detection of inherent biases in the selection, which may harm both the workshop's success as well as the larger community.
```
Comment (arokem) [inline paper.md]: True for Neurohackweek.

File: paper.md [hunk 4]
```diff
@@ -33,120 +33,183 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
...
+
+However, there are also drawbacks to limiting the theme of a hack week to a single field. Homogeneity may be a disadvantage if it leads to _group think_ and inhibits new, creative solutions. Within a field, the chance of participants being familiar with each other may be larger, leading to the formation of cliques and insufficient mixing. These disadvantages may be mitigated by organizing a hack week around a technique (e.g. Gaussian Processes) or modality (e.g. imaging) instead. In these case, building a shared vocabulary and understanding the major data analysis problems in each field is a crucial task, and should be allocated sufficient time in form of introductory talks or tutorials to foster cross-disciplinary collaboration. On the other hand, cross-disciplinary hack weeks around a specific technique or type of data set allow methods developed in a specific field to diffuse into other subjects and therefore help avoid duplication of method development efforts. 
+
+**[Can someone write something about the XD meetings in Berkeley and UW??]**
```
Comment (arokem) [inline paper.md]: I can do that. Will make a PR on top of this one...

File: paper.md [hunk 5]
```diff
@@ -33,120 +33,183 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
...
 
 ### What to do before
 
-- WAY BEFORE: get IRB approval to track and study participants! This is a hard one, but very, very valuable.
+#### Obtaining an IRB to track and study participants
+For a large fraction of the hack weeks we have previously organized, studying the development of a hack week, its participants and outcomes was a stated goal of the workshop  **[GEO/NEURO: Is this true for you?]**. We included social scientists interested in scientific collaboration in the design process and organization, who both studied the workshop itself and gave valuable feedback about its outcomes. However, in order to be able to track and study participants, approval from Internal Review Boards is a must for workshops organized in the United States. This should be done as early as possible, and having assistance from researchers in the social sciences is invaluable, in particular for hack weeks in fields that generally don't involve human subject research. Ideally, participants should consent to be contacted at the application stage, which opens up the potential of tracking both participants and non-participants and allows for evaluation of the hack week.
```
Comment (arokem) [inline paper.md]: Yes. We had folks from the local Data Science Studies group observing last year, and I hope they'll join in this year as well.

**Review by arokem — COMMENTED** [review]
I had just a few small comments and affirmations here. It looks great. I am going to make a PR on top of this with a few details about ImageXD.

File: paper.md (hunk 2 above)
Comment (dhuppenkothen) [reply paper.md]: Which is this referring to? The `Astrohack week` or the `hack week`? The former is a typo, the other based on my understanding that merging two words into one is less accepted in English than it is in German ... ;)

File: paper.md (hunk 1 above)
Comment (dhuppenkothen) [reply paper.md:43]: Fingers crossed it won't take that long. 

In other news: I've been co-organizing Python in Astronomy this year, which shares many features with the hack week model, including tutorials, unconference sessions, sprints and a hack day. I've referred to it a couple of times in the text, but I'm curious if anyone minds to include it more formally (e.g. in the figure)?

**Comment by karthik** [comment]
Still hoping to review soon. Sorry for the delays.

**Comment by dhuppenkothen** [comment]
As discussed on Tuesday, I'm going to merge this as a draft, and we can start editing from there.

**Pull request merged**